pkg/registry/populator.go

```
@@ -63,14 +65,21 @@ func (i *DirectoryPopulator) Populate(mode Mode) error {
 	return nil
 }
 
-func (i *DirectoryPopulator) globalSanityCheck(imagesToAdd []*ImageInput) error {
+func (i *DirectoryPopulator) globalSanityCheck(imagesToAdd []*ImageInput, mode Mode) error {
+
+	if i.overwrite && mode != ReplacesMode {
+		return OverwriteErr{ErrorString: "overwrite-latest flag is only supported in Replaces mode"}
+	}
```

should we also return an error when `len(imagesToAdd) > 1 && i.overwrite`?